docker-compose.yml

```
@@ -93,10 +106,10 @@ services:
     env_file:
       - ./client-e2e/.env
     volumes:
-      - ./client-e2e:/usr/src/app
+      - ./client-e2e/test:/usr/src/app/client-e2e/test
```

I suggest we stick to mounting the root instead of the test folder:
- The user might want to add or change scripts in the `scripts` folder
- If the user `npm install`s a new dependency inside the container, the updated `package.json`/`package-lock.json` will be easy to commit

```suggestion
      - ./client-e2e:/usr/src/app/client-e2e
```

---

I don't have much context here, but if it's just a speed issue it would be best to keep "dev" and "CI" as close as possible. That is, if CI is a bit slower, but devs can use the defaults that's a "win".

(And, yeah, in my limited "using Docker for dev" experiences it's best to have live / container-mounted things as @JustusFT is asking)

---

We might not need a completely separate docker-compose. We can try use an [override](https://docs.docker.com/compose/extends/#multiple-compose-files) if we need the speedup. The existing docker-compose.yml would be the base, but in the override you can remove the volume mounts.

We can also add an argument to the `start` function so it sets `livereload` to `false` when running `gulp start` but not `gulp watch`.

---

Rollback some changes. Seems now is running with mounted directory.

---

Wider context here: make it work, _then_ make it faster (if required) ;)

Since the main user of the environment is "other devs" (right?) the features they need should be highest priority. If it turns out to be "too slow" in CI, that's the time to look for optimizations.